src/Heading.js

```diff
@@ -0,0 +1,11 @@
+const Heading = () => (
+  <header className="App-header">
+    <img
+      className="App-logo"
+      src="https://img.freepik.com/premium-vector/hotel-logo-simple-illustration_434503-736.jpg?w=2000"
+    />
+    <span className="App-title">CYF Hotel</span>
+  </header>
```

Comment:
Perfect, small improvement could be to add the `imageSrc` and `spanText` as props - but it's also fine to leave as-is if they're never going to be reused.